server/controllers/routes/admin/cohort/deleteCohort.js

```diff
@@ -0,0 +1,17 @@
+const deleteCohort = require('../../../../database/queries/cohort/deleteCohort');
+
+const cohortDelete = async (req, res, next) => {
+  try {
+    await deleteCohort(req.params.cohortId);
+    res.json({
+      StatusCode: 200,
+      data: { message: 'Cohort deleted successfully' },
+    });
+  } catch (err) {
+    next(err);
+  }
+};
+
+module.exports = {
```

Comment:
@MohammedAlghazali This isn't a super-important thing to fix, but to be consistent with exports you could make this a default export (`module.exports = cohortDelete`) since it will probably be the only export in this file.  That would be similar to how you exported in `queries/cohort/deleteCohort.js` below.
If you use a consistent rule like this, where every file that has only a single exported function uses a default export, that can make it more predictable / easy for others who are trying to import your functions in their files.

Reply:
yes, i must be careful about this point :thumbsup: